src/WaveTrack.cpp

```diff
@@ -983,18 +1138,19 @@ void WaveTrack::ClearAndPaste(double t0, // Start of time to clear
             double st;
             double et;
 
-            st = clip->GetStartTime();
-            et = clip->GetEndTime();
+            st = clip->GetPlayStartTime();
+            et = clip->GetPlayEndTime();
 
             // Scan the cuts for any that live within this clip
             for (auto it = cuts.begin(); it != cuts.end();) {
                WaveClip *cut = it->get();
-               double cs = cut->GetOffset();
+               //cutlines in this array were orphaned previousely
+               double cs = cut->GetSequenceStartTime();
 
                // Offset the cut from the start of the clip and add it to
                // this clips cutlines.
                if (cs >= st && cs <= et) {
-                  cut->SetOffset(warper->Warp(cs) - st);
+                  cut->SetSequenceStartTime(warper->Warp(cs) - st);
```

Comment:
I thought there might be a bug with cut lines, but I read all again carefully and no longer think so.  Good job